expressions/control-structures.md

```diff
@@ -121,15 +121,15 @@ Just like `if` expressions `while` is also an expression. The value returned is
 
 __But what if the condition evaluates to false the first time we try, then we don't go round the loop at all?__ In Pony `while` expressions can also have an `else` block. In general, Pony `else` blocks provide a value when the expression they are attached to doesn't. A `while` doesn't have a value to give if the condition evaluates to false the first time, so the `else` provides it instead.
 
-__So is this like an else block on a while loop in Python?__ No, this is very different. In Python the `else` is run when the `while` completes. In Pony the `else` is only run when the expression in the `while` isn't.
+__So is this like an else block on a while loop in Python?__ No, this is very different. In Python, the `else` is run when the `while` completes. In Pony the `else` is only run when the expression in the `while` isn't.
 
 ### Break
 
-Sometimes you want to stop part-way through a loop and give up altogether. Pony has the `break` keyword for this and it is very similar to its counterpart in languages like C++, C# and Python.
+Sometimes you want to stop part-way through a loop and give up altogether. Pony has the `break` keyword for this and it is very similar to its counterpart in languages like C++, C#, and Python.
 
 `break` immediately exits from the innermost loop it's in. Since the loop has to return a value `break` can take an expression. This is optional and if it's missed out the value from the `else` block is returned.
 
-Let's have an example. Suppose you want to go through a list of names you're getting from somewhere, looking for either "Jack" or "Jill". If neither of those appear you'll just take the last name you're given and if you're not given any names at all you'll use "Herbert".
+Let's have an example. Suppose you want to go through a list of names you're getting from somewhere, looking for either "Jack" or "Jill". If neither of those appears you'll just take the last name you're given and if you're not given any names at all you'll use "Herbert".
```

Comment:
It should be "neither of those appear", for the same reason that you use "each is" rather than "each are". Also add a comma after "appear".